rest/rest/src/main/java/io/syndesis/rest/v1/handler/connection/CustomConnectorHandler.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.rest.v1.handler.connection;
+
+import java.util.Collections;
+import java.util.List;
+
+import javax.persistence.EntityNotFoundException;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import io.syndesis.connector.generator.ConnectorGenerator;
+import io.syndesis.dao.manager.DataManager;
+import io.syndesis.model.connection.Connector;
+import io.syndesis.model.connection.ConnectorTemplate;
+import io.syndesis.model.connection.ConnectorSettings;
+import io.syndesis.rest.v1.operations.Violation;
+
+import org.springframework.context.ApplicationContext;
+
+public final class CustomConnectorHandler {
+
+    private final ApplicationContext applicationContext;
+
+    private final String connectorTemplateId;
+
+    private final DataManager dataManager;
+
+    /* default */ CustomConnectorHandler(final String connectorTemplateId, final DataManager dataManager,
+        final ApplicationContext applicationContext) {
+        this.connectorTemplateId = connectorTemplateId;
+        this.dataManager = dataManager;
+        this.applicationContext = applicationContext;
+    }
+
+    @POST
+    @Path("/info")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    @ApiOperation("Creates a new Connector based on the ConnectorTemplate identified by the provided `connector-template-id` and the data given in `customConnector`")
```

Comment:
adapt documentation.

Best make a search through the project with 'customConnector' and change everything to `connectorSettings` (could be that I missed something, too)